src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -525,6 +527,27 @@ internal Collection<FunctionDescriptor> ReadFunctions(List<FunctionMetadata> met
             return functionDescriptors;
         }
 
+        private List<FunctionMetadata> ApplyMaximumFunctionCount(List<FunctionMetadata> metadatas)
```

Comment:
`functions` is probably a clearer name than `metadatas`. Besides, `metadatas` is not a valid plural form for `metadata`. ;)